File: examples/installation/osx/objc/DynamicExample/DynamicExampleTests/DynamicExampleTests.m
//
//  DynamicExampleTests.m
//  DynamicExampleTests
//
//  Created by JP Simard on 5/6/15.
//  Copyright (c) 2015 Realm. All rights reserved.
//

#import <Cocoa/Cocoa.h>
#import <XCTest/XCTest.h>
#import "AppDelegate.h"

@interface DynamicExampleTests : XCTestCase
@end

@implementation DynamicExampleTests

- (void)testExample {
    XCTAssertTrue([MyModel isSubclassOfClass:[RLMObject class]]);
}

@end

File: examples/installation/osx/swift-2.0/DynamicExample/DynamicExampleTests/DynamicExampleTests.swift
//
//  DynamicExampleTests.swift
//  DynamicExampleTests
//
//  Created by JP Simard on 6/25/15.
//  Copyright (c) 2015 Realm. All rights reserved.
//

import Cocoa
import XCTest
import DynamicExample
import RealmSwift

class DynamicExampleTests: XCTestCase {
    func testExample() {
        // This is an example of a functional test case.
        XCTAssertNotNil(MyModel() as AnyObject is Object)
    }
}
